tests/simulator/test_advance.py

```diff
@@ -237,6 +240,106 @@ def test_field_coarsening_via_subcycles(self, refinement_boxes):
         self._test_field_coarsening_via_subcycles(dim, interp_order=1, refinement_boxes=refinement_boxes)
 
 
+
+
+    def _test_field_level_ghosts_via_subcycles_and_coarser_interpolation(self, ndim, interp_order, refinement_boxes, refinement_ratio=2):
+        """
+          This test runs two virtually identical simulations for one step.
+            L0_datahier has no refined levels
+            L0L1_datahier has one refined level
+
+            This is done to compare what would be the uncoarsened L0 values in L0L1_datahier
+              if it had no refined level to the level field ghost of L1 of L0L1_datahier
+        """
+
+        print("test_field_coarsening_via_subcycles for dim/interp : {}/{}".format(ndim, interp_order))
+
+        from tests.amr.data.field.refine.test_refine_field import refine
+        from tests.core.numerics.interpolator.interpolator_test import interpolate_datasets
+        from pyphare.pharein import global_vars
+
+        time_step_nbr = 1 # after first advance, simulations diverge
```

Comment:
"diverge" has a meaning (in numerical math) that the simulation goes crazy (results diverge exponentially in time). I know this is not what you mean here where simulations literally diverge from one another. Maybe we should add "from one another" in the comment.

Maybe we should say a bit more in the docstring, like saying that these two simulations are expected to have identical L0 for the first substep only before any coarsening occurs on the one that's refined, hence the only step that's simulated.